platforms/software/resources-http/src/test/groovy/org/gradle/internal/resource/transport/http/HttpResourceAccessorTest.groovy

```diff
@@ -16,11 +16,15 @@
 
 package org.gradle.internal.resource.transport.http
 
+import groovy.util.logging.Slf4j
 import org.apache.http.StatusLine
 import org.apache.http.client.methods.CloseableHttpResponse
+import org.gradle.api.internal.DocumentationRegistry
+import org.gradle.internal.resource.ExternalResource
 import org.gradle.internal.resource.ExternalResourceName
 import spock.lang.Specification
 
+@Slf4j
```

Comment:
❓ generally, we don't use this annotation, and I don't see a reason to use it in tests

Reply:
It was used when running unit test locally, I'll remove it before merge.